tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -28,16 +28,18 @@ sub quit_firefox {
 }
 
 sub run {
-    my ($self) = shift;
+    my ($self) = @_;
 
     # Define FIPS password for firefox, and it should be consisted by:
     # - at least 8 characters
     # - at least one upper case
     # - at least one non-alphabet-non-number character (like: @-.=%)
     my $fips_password = 'openqa@SUSE';
 
-    # Turn off screensaver before launch firefox in order to avoid the screensaver block
-    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
```

Comment:
yes, this error https://openqa.suse.de/tests/3022911#step/firefox_nss/4 *is* introduced by `turn_off_gnome_screensaver`
, and you explanation makes sense. 
So, this invalid line can be deleted.

But I can not understand why the case `seahorse_sshkey` can pass after deleting the above line and
maybe adding line `select_console 'root-console';` make the "seahorse_sshkey" pass.

Whatever, all the changes are no harm and work.

Reply:
1. We added this function call to be a workaround before to avoid the screensaver lock while launching the firefox, but actually turn_off_gnome_screensaver is not working at all in the case, thus our solution was not a real fixing here.

2. After adding this function call, there is no console which can accept the code to turn off gnome screensaver, thus it just typed "into the void" of the desktop session, but openQA is still waiting for a wait_serial string matching, it causes error I observed in the log. But afterward it does not lead to the case Fail, the firefox_nss case is still Passed in this situation.

3. On the other hand, we can use select_console 'root-console' or xterm to type the command script_run 'gsettings set org.gnome.desktop.session idle-delay 0'; 
   e.g., add x11_start_program('xterm') and then run turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome'), I think it can also fixed the problem.

4. After testing, it is still Fail after adding line select_console 'root-console' in "seahorse_sshkey" too. Maybe it should be a backend problem.

To sum up, in fact, we don't need to really turn off the screen saver at all, I think the real fixing is to revert the turn_off_gnome_screensaver function call.

Reply:
for `4.` your verify run is PASS not "Fail".

Reply:
Yes, the verification run is PASSED now. :) 
I mean if I only add **select_console 'root-console'** in `seahorse_sshkey`, but did not revert the **turn_off_gnome_screensaver** call in `firefox_nss` , the case `seahorse_sshkey` is still Fail.

Reply:
It does not explain why **turn_off_gnome_screensaver** make following `select_console` failed. If we want to dive deeper we should check the code which implement **turn_off_gnome_screensaver** in x11utils. But as a work around it is acceptable to simply remove it from the code.